changelogs/fragments/rpm_ostree_package_manager.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- Added a functionality to manage overlay packages using rpm_ostree command.
```

Comment:
Changelog fragments are not needed for new modules and plugins (except `filter` and `test` plugins).